pages/collection/[id]/play/index.js

```diff
@@ -234,26 +238,26 @@ export default function PlayMode({ collections, flashCards }) {
               )}
             </StyledCardContainer>
             <StyledFooter>
-              <button onClick={handleNext}>❌</button>
+              <StyledX onClick={handleNext}></StyledX>
               <p aria-label="page-counter">
                 {currentPage + 1} / {totalPages}
               </p>
-              <button onClick={handleCorrect}>✅</button>
+              <StyledCheck onClick={handleCorrect}></StyledCheck>
             </StyledFooter>
           </>
         )}
```

Comment:
Same here: This is an interactive element, therefore I strongly suggest using a button here and placing the icon inside the button.